StripeFinancialConnections/StripeFinancialConnections/Source/FinancialConnectionsSheet.swift

```diff
@@ -70,7 +70,7 @@ final public class FinancialConnectionsSheet {
     /// Completion block called when the sheet is closed or fails to open
     private var completion: ((HostControllerResult) -> Void)?
 
-    private var hostController: HostController?
+    private(set) var hostController: HostController?
```

Comment:
@kgaidis-stripe I'm curious of your thoughts on this change. I did this to get access to the presented view controller from the `FinancialConnectionsSheet`. If you know of another way (without exposing this property) I'm happy to revise!

Reply:
we should be good! its `internal` anyway so if we need to access it from tests it is what it is 👍